src/main/ruby/core/stat.rb

```diff
@@ -134,13 +134,11 @@ def chardev?
     end
 
     def dev_major
-      major = Truffle::POSIX.major dev
-      major < 0 ? nil : major
```

Comment:
I can't readily tell from here. Is the caller just now handling a negative return value?

Reply:
These macros return `unsigned int` so it's just impossible to have a negative return value.

Reply:
To be precise, MRI only returns `nil` when the minor/major macros are not available.
But our 3 platforms have these macros so we don't need to handle that case.